backend/src/main/java/pl/seahawks/vacationes/domain/model/Holiday.java

```diff
@@ -0,0 +1,35 @@
+package pl.seahawks.vacationes.domain.model;
+
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import lombok.Setter;
+
+import javax.persistence.*;
+import java.math.BigDecimal;
+import java.time.LocalDate;
+
+@Getter
+@Setter
+@NoArgsConstructor
+@Entity
+public class Holiday {
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private Long id;
+    private String carrierName;
+    @ManyToOne
+    private Location destination;
+    @OneToOne
+    private Flight flight;
+    private BigDecimal price;
+    private Integer duration;
+    private LocalDate start;
+    private LocalDate end;
+    private String url;
+    private String pictureUrl;
+    @Enumerated(EnumType.STRING)
+    private OfferType offerType;
+    @Enumerated(EnumType.STRING)
+    private OfferPackage offerPackage;
+
```

Comment:
Pusta linia, można usunąć. Jak używasz intelij to możesz zainstalować plugin google-java-format do formatowania kodu

Reply:
Używam w sumie checkstyle od javastart, ale on faktycznie nie wylapuje pustych lini. Ten z googla wylapuje za to troche za duzo :P no nic nastepnym razem bede mial na oku puste linie. Dzięki!